.circleci/config.yml

```diff
@@ -157,14 +157,22 @@ commands:
       - run:
           name: Check for any unrecorded changes in our dependency trees
           command: |
-            pip3 install requests
+            pip3 install --require-hashes -r ./tools/requirements.txt
             rustup install nightly
             cargo metadata --locked > /dev/null
             python3 ./tools/dependency_summary.py --check ./DEPENDENCIES.md
             python3 ./tools/dependency_summary.py --all-android-targets --package megazord --check megazords/full/DEPENDENCIES.md
             python3 ./tools/dependency_summary.py --all-ios-targets --package megazord_ios --check megazords/ios/DEPENDENCIES.md
             python3 ./tools/dependency_summary.py --all-android-targets --package fenix --check megazords/fenix/DEPENDENCIES.md
             python3 ./tools/dependency_summary.py --all-android-targets --package lockbox --check megazords/lockbox/DEPENDENCIES.md
+  sync-bugzilla-to-github:
+    steps:
+      - checkout
+      - run:
+          name: Mirror Bugzilla issues into GitHub
+          command: |
+            sudo pip3 install --require-hashes -r ./tools/requirements.txt
```

Comment:
NIT: Could this path `./tools/requirements.txt` be too generic at the moment as only one CI task is using these dependencies whereas others tasks are defining them another way?

Reply:
Do you mean this should be more like "bugzilla-sync-requirements.txt" so that other tools could use other requirements files?